### PR TITLE
Enable

### PR DESCRIPTION
**Pull request by damongreen123**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "nodejs"
+def product = "em"
+def component = "icp-api"
```
Comment (yogesh-hullatti) [inline Jenkinsfile_CNP:7]: Should the component name just be - "icp" ? Which will result in the chart name as "em-icp"

File: charts/em-icp/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: em-icp
+home: https://github.com/hmcts/rpa-em-icp-api
```
Comment (yogesh-hullatti) [inline charts/em-icp/Chart.yaml:3]: Do we need to move the github repo as we did for others? i.e. From rpa-em-icp-api to em-icp-api ?

File: infrastructure/prod.tfvars [hunk 3]
```diff
@@ -0,0 +1,3 @@
+dns_server="10.96.75.254"
+capacity = "2"
```
Comment (adusumillipraveen) [inline infrastructure/prod.tfvars:2]: this shouldn't be needed.

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+@Library("Infrastructure")
+
+def type = "nodejs"
+def product = "em"
+def component = "icp"
+
+def secrets = [
```
Comment (adusumillipraveen) [inline Jenkinsfile_CNP:9]: Can't you avoid using s2s secret from s2s vault. I am guessing you have plans to copy to ur vault anyway?

**Pull request merged**